src/components/Avatar.js

```diff
@@ -1,9 +1,8 @@
-// Avatar component takes person object as props
-// returns an img element with className 'Avatar'
-// img element should have src and alt attributes
-
 export default function Avatar(props) {    
     const { person } = props;
-
-    // return ...
-}
\ No newline at end of file
+    return(
+        <div>
```

Comment:
if it is not necessary do not add extra html elements to the DOM (do some reading about "div hell" )